src/app/video-courses/filter-by-title.pipe.ts

```diff
@@ -0,0 +1,16 @@
+import { Pipe, PipeTransform } from '@angular/core';
+import { VideoCourseItem } from './video-course-item.model';
+
+@Pipe({
+  name: 'filterByTitle'
+})
+export class FilterByTitlePipe implements PipeTransform {
+
+  transform(allCourses: VideoCourseItem[], searchString: string): VideoCourseItem[] {
```

Comment:
Make less specific, you can filter any entity by title field, not only `VideoCourseItem`. For an input value type use something like `Partial<{title: string}>` as inside pipe you care only about `title` field.

Reply:
Done